client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1127,6 +1127,15 @@ Undocumented.prototype.readA8C = function( query, fn ) {
 	return this.wpcom.req.get( '/read/a8c', query, fn );
 };
 
+Undocumented.prototype.readConversations = function( query, fn ) {
```

Comment:
> maybe with tree-shaking it'll be ok. 

less about how it compiles, more about how if you want to look for api functions you can always check a single file + CMD F in it